components/camera/transformpipeline/classifier.go

```diff
@@ -28,7 +27,7 @@ type classifierConfig struct {
 
 // classifierSource takes an image from the camera, and overlays labels from the classifier.
 type classifierSource struct {
-	stream             gostream.VideoStream
+	src                camera.VideoSource
```

Comment:
why can't this take a camera?

Reply:
https://github.com/viamrobotics/rdk/pull/4487#discussion_r1842930728